draft-ietf-httpbis-http2-secondary-certs.md

```diff
@@ -770,6 +844,13 @@ its control in order to present the compromised certificate. As recommended in
 {{?RFC8336}}, clients opting not to consult DNS ought to employ some alternative
 means to increase confidence that the certificate is legitimate.
 
+One such means is the Allowed Primary Domain certificate extension defined in
+{extension}. Clients SHOULD require that server certificates presented via this
+mechanism contain the Allowed Primary Domain extension and require that a
+certificate previously accepted on the connection (including the certificate
+presented in TLS) lists one of the Allowed Primary Domains in the Subject field,
+the Subject Alternative Name extension, or the Affiliated Domains extension.
```

Comment:
This isn't very clear to me.  The point here is that you can allow either mechanism.